data/wiki/hello-world.md

```diff
@@ -582,8 +557,8 @@ Build this on Unix in the same way as the previous examples:
 ```bash
 $ cd device-usage/block
 $ mirage configure -t unix
-$ make depend
-$ make
+$ make depends
+$ make build
```

Comment:
I don't know if `make build` or `dune build` is the most useful to mention here. Maybe just saying that after `make depend(s)` has run, the workspace is setup and can just be built using `dune` (no need to interact with the `mirage` tool nor the `Makefile` file again).

Reply:
Note: we should probably hide this `make depends` into `mirage depends` at one point (and/or wait for `dune` to be able to deal with this directly as well)

Reply:
Yeah: I think it'd be lovely to be able to hide/remove the make invocations if possible.

On a related note, do you think the generated build artifacts during "configure" could end up being moved into `_build` at some point? It'd be cool to be able to run all the different configurations, and have those only set up different sub-trees in `_build` without having all the intermediate files changing the work tree.

Reply:
> Maybe just saying that after make depend(s) has run, the workspace is setup and can just be built using dune (no need to interact with the mirage tool nor the Makefile file again).

That removes a level of indirection, but introduces a third command, leaving the user with `mirage`, `make`, and `dune`, instead of just `mirage` and `make`. Is that trade off preferable?

Reply:
> On a related note, do you think the generated build artifacts during "configure" could end up being moved into _build at some point? It'd be cool to be able to run all the different configurations, and have those only set up different sub-trees in `_build` without having all the intermediate files changing the work tree.

Sounds like a good suggestion to me, would you mind opening an issue on github.com/mirage/mirage about it?

> That removes a level of indirection, but introduces a third command, leaving the user with `mirage`, `make`, and `dune`, instead of just `mirage` and `make`. Is that trade off preferable?

I'm not sure. In the end I'd prefer to remove `make` from the process, since we won't get rid of `mirage` soon, and for more sophisticated things we need to interact with `dune` directly anyways. But indeed for the tutorial, I'd stick with `mirage` and `make`, and avoid `dune`. :)

Reply:
> Sounds like a good suggestion to me, would you mind opening an issue on github.com/mirage/mirage about it?

Happy to do so!